READMeCHALLENGE.md

```diff
@@ -0,0 +1,73 @@
+# 🧪 QA / Test Automation Engineer Challenge – New guide for AutomationExercise.com
```

Comment:
The README is comprehensive but could be improved with:
- Screenshots of test execution
- More detailed setup instructions
- Troubleshooting section
- Test coverage report